islandora.module

```diff
@@ -1962,11 +1962,11 @@ function islandora_islandora_metadata_display_info() {
  * Implements hook_islandora_datastream_access().
  */
 function islandora_islandora_datastream_access($op, AbstractDatastream $datastream, $user) {
-  module_load_include('inc', 'islandora', 'includes/utilities');
-  $result = islandora_namespace_accessible($datastream->parent->id) && user_access($op, $user);
+  $result = islandora_islandora_object_access($op, $datastream->parent, $user);
```

Comment:
@rosiel why is the hook called the islandora_ specific hook and not the whole islandora_object_access wrapper/callback? That leaves any other user/module contributed access callbacks that implement the hook_islandora_object_access out.